features/steps/run.py

```
@@ -12,10 +12,21 @@ def run(context: Context, command: str):
+    env["LC_ALL"] = "en_US.UTF-8"
+    if hasattr(context, "env"):
+        env.update(context.env)
     if hasattr(context, "home_dir"):
         # Override the HOME variable
         env["HOME"] = context.home_dir
 
+    # In case of Windows, we need to ensure that SYSTEMROOT and COMSPEC variables are set
+    system_root = os.environ.get("SYSTEMROOT", "C:\\Windows")
+    comspec = os.path.join(system_root, "System32", "cmd.exe")
+    os.environ["SYSTEMROOT"] = system_root
+    os.environ["SystemRoot"] = system_root
```

Ruff (0.8.2)

Use capitalized environment variable `SYSTEMROOT` instead of `SystemRoot`

Replace `SystemRoot` with `SYSTEMROOT`

(SIM112)